sdks/python/apache_beam/pipeline_test.py

```diff
@@ -528,6 +528,58 @@ def f_c(x):
       p.run().wait_until_finish()
     assert count_b == count_c == 4
 
+  def test_no_partial_writeouts(self):
+    from collections import defaultdict
+    from apache_beam.runners.direct.transform_evaluator import _TransformEvaluator
+    from apache_beam.runners.direct.transform_evaluator import _GroupByKeyOnlyEvaluator
+    from apache_beam.runners.direct.evaluation_context import _ExecutionContext
```

Comment:
<!--thread_id:cc_139172688_t; commit:af94b7b7a8c0ac0ada222884101ec10127f5b234; resolved:0-->
<!--section:context-quote-->
> **mariapython** wrote:
> Done.

<!--section:body-->
I don't see the change here.